project/Extra.scala

```diff
@@ -8,11 +8,26 @@ import bintray.BintrayKeys.{bintrayRepository, bintrayOrganization}
 
 object Extra {
 
-  val sbtPluginSettings = ScriptedPlugin.scriptedSettings ++ Seq(
+  val sbtPluginSettings = ScriptedPlugin.scriptedSettings.filterNot(
+      _.key.key.label == libraryDependencies.key.label) ++ Seq(
+      // https://github.com/sbt/sbt/issues/3325
```

Comment:
I am not sure I understand what's going on, here? Is the workaround implemented here sufficient to get scripted to work with 1.0.0-RC2? If yes, why doesn't .travis.yml run scripted for the plugin under 1.0.0-RC2? If not, what is the purpose of these settings here?